email_recovery/pubspec.yaml

```diff
@@ -43,7 +43,6 @@ dev_dependencies:
 
 # The following section is specific to Flutter.
 flutter:
-
   uses-material-design: true
```

Comment:
please revert it

Reply:
Done

Reply:
Looks like you forgot to revert it. @tddang-linagora